runtime/compiler/runtime/SymbolValidationManager.cpp

```diff
@@ -1252,7 +1252,12 @@ TR::SymbolValidationManager::validateArrayClassFromComponentClassRecord(uint16_t
    if (isDefinedID(componentClassID))
       {
       TR_OpaqueClassBlock *componentClass = getClassFromID(componentClassID);
-      return validateSymbol(arrayClassID, _fej9->getArrayClassFromComponentClass(componentClass));
+      if (validateSymbol(arrayClassID, _fej9->getArrayClassFromComponentClass(componentClass)))
+         return true;
+
+      TR_OpaqueClassBlock *nullRestrictedArray = TR::Compiler->om.areFlattenableValueTypesEnabled() ?
+                                                   _fej9->getNullRestrictedArrayClassFromComponentClass(componentClass) : NULL;
```

Comment:
Is it necessary to test `TR::Compiler->om.areFlattenableValueTypesEnabled()` here?  Shouldn't calling that method return NULL if is flattenable value tyeps are not enabled?

Reply:
Addressed in de9f0d5. Ready for another review